### DISCUSSION TITLE
Any chance to get v7.20 released for @babel/preset-typescript?

### DISCUSSION THREAD

**Discussion by jraoult** [babel/babel#15422] [Releases] [answered]
Is there a reason for the `@babel/preset-typescript` v7.20 not being released yet @nicolo-ribaudo ? 

I'm keen on the added syntax support in https://github.com/babel/babel/pull/14211, and it seems it was part of the 7.20 milestone https://github.com/babel/babel/milestone/54?closed=1 and expected by this comment: https://github.com/babel/babel/pull/14211#issuecomment-1292544846. I couldn't find any discussion explaining why the `core` package was released about two weeks ago but not the TS plugin.

**Comment by nicolo-ribaudo** [comment, marked as answer]
That change only needed to be implemented in `@babel/parser` and `@babel/plugin-transform-typescript`: make sure that they are up to date, and you'll be fine :)

With Yarn you can do `yarn up -R '@babel/*'`, probably npm and pnpm have a similar command (worst case, delete those two packages from your lockfile and re-run `npm install`/`pnpm install`).

**Reply by jraoult** [reply]
@nicolo-ribaudo Thanks, I see how I got confused. I was expecting a v7.20 for the preset that would pull the correct transform version for me. I'm currently only explicitly depending on `@babel/preset-typescript`. I was under the impression that the preset was the [recommended](https://babeljs.io/docs/en/babel-preset-typescript) way to [install](https://www.typescriptlang.org/docs/handbook/babel-with-typescript.html) what is necessary to get Babel to work for TS. I'm I wrong to think that if a new version of the TS transform plugin is released, so should the preset be?

BTW, the force upgrade worked for me, I suspect because it forces the upgrade of the transitive dependencies as well
